content/code-signing-yaml/signing-macos.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Signing macOS apps
+description: How to set up macOS code signing in codemagic.yaml
+weight: 2
+aliases: '../code-signing-yaml/signing'
```

Comment:
Why is this alias here for a new file? Can we remove it?

Reply:
it's just a copy-paste:) removed in https://github.com/codemagic-ci-cd/codemagic-docs/pull/320/commits/631fb2e517b63ef00d3ef2a2d6a09183576fcca4#diff-ed26953a6de599789eded1bb99c9cbe95db2ca812b27a37ae43c5a8499c1a38dL5